java-engine/src/main/java/io/getunleash/engine/CustomStrategiesEvaluator.java

```diff
@@ -63,13 +63,15 @@ public void loadStrategiesFor(String toggles) {
         }
     }
 
-    private List<MappedStrategy> getFeatureStrategies(FeatureDefinition feature) {
+    List<MappedStrategy> getFeatureStrategies(FeatureDefinition feature) {
         List<MappedStrategy> mappedStrategies = new ArrayList<>();
         int index = 1;
         for (StrategyDefinition strategyDefinition : feature.strategies) {
-            IStrategy impl =
-                    Optional.ofNullable(registeredStrategies.get(strategyDefinition.name))
-                            .orElseGet(() -> alwaysFalseStrategy(strategyDefinition.name));
+            if (builtinStrategies.contains(strategyDefinition.name)) {
+                continue;
```

Comment:
This check is actually the salient point here. If we find our strategy matches a built in strategy, just continue